poller-wrapper.py

```diff
@@ -86,7 +87,6 @@ def get_config_data():
     db_port = int(config['db_host'].rsplit(':')[1])
 else:
```

Comment:
Probably a hangup from when we used mysql_connect which I think did/does accept :port